runtime/lib/app-runtime.js

```diff
@@ -1124,7 +1054,10 @@ AppRuntime.prototype.login = _.singleton(function login (options) {
           return Object.assign({}, config)
         }
         this.component.wormhole.setClient(this.cloudApi.mqttcli)
-        this.onLoadCustomConfig(_.get(config, 'extraInfo.custom_config', ''))
+        if (config && typeof config === 'object' && config.extraInfo && typeof config.extraInfo === 'object' &&
```

Comment:
Could be replaced by `typeof _.get(config, 'extraInfo.custom_config) === 'string'`

Reply:
ok, updated